CMakeLists.txt

```diff
@@ -80,6 +80,12 @@ IF (WITH_SWDB)
     pkg_check_modules(SQLite3 sqlite3 REQUIRED)
 ENDIF()
 
+IF (WITH_SWDB)
+   set (SWDB_PKGCONFIG_DEPS "sqlite3")
+ELSE()
+   set (SWDB_PKGCONFIG_DEPS "")
+ENDIF()
+
```

Comment:
This will most likely go away when we merge new swdb code.
Libdnf will always require sqlite3, because history is integral part of YUM / DNF now and this functionality will move into libdnf so it's available for microdnf, PackageKit, etc.

Reply:
We currently have the conditional for SWDB, so I've wrapped it accordingly...